client/src/components/common/Typography/style.js

```diff
@@ -0,0 +1,14 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const Theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: 'rgb(237, 109, 35)',
+    },
+    secondary: {
+      main: 'rgb(59,90,111)',
+    },
+  },
+});
```

Comment:
here you can create the styles you want for every type of typography you expected to use in our project !
for all headings `h1`, `h2`, ....., `subtitle1`, .... , `body1` , ... 
you can add here , color , fontSize, fontWeight, fontfamily & others as we need & looks in `designs` again
```suggestion
const useStyles = makeStyles({
  root: {
    width: "100%",
    maxWidth: 500
  },
  h1: {
     fontSize:24,
    color: "red"  //for example
  }
});
```